app/views/vm_common/_live_migrate.html.haml

```diff
@@ -52,6 +52,23 @@
                   'paging_div_buttons_state_enabled' => true,
                   'paging_div_buttons_id'            => "angular_paging_div_buttons",
                   'paging_div_buttons_type'          => "Submit"}
+- unless @explorer
+  %table{:width => '100%'}
+    %tr
+      %td{:align => 'right'}
+        #buttons_on
```

Comment:
@mansam you will also need 
```
button_off 
``` 
code to handle form buttons when submit button is disabled on the screen, so that submit button is displayed when the forms are loaded initially.

see screenshots below for 
initial load of reconfig screen
![screenshot from 2016-06-09 15 00 03](https://cloud.githubusercontent.com/assets/3450808/15942698/7a0fc02e-2e53-11e6-8a3c-047856e1302a.png)

after change has been made to reconfig form
![screenshot from 2016-06-09 15 00 18](https://cloud.githubusercontent.com/assets/3450808/15942694/76f6dd0a-2e53-11e6-8641-766f753e7c69.png)


 screen that is working correctly: